resourceIndexer/coreStagingS3.js

```diff
@@ -308,13 +308,9 @@ function validIntermediate(id, date, objects) {
         })
         .map((s3object) => {
           const filename = s3object.key.split('/').pop();
-          let url = s3object.versionId ?
+          const url = s3object.versionId ?
             `${s3object.baseUrl}?versionId=${encodeURIComponent(s3object.versionId)}` :
             s3object.baseUrl;
-          /* we prefer to serve objects on the nextstrain-data bucket via the cloudfront-ed
-          data.nextstrain.org domain rather than nextstrain-data.s3.amazonaws.com. This
-          continues to work with versionId queries */
-          url = url.replace('https://nextstrain-data.s3.amazonaws.com', 'https://data.nextstrain.org');
```

Comment:
[discussion]

The server has 2 APIs to provide data to /pathogen/files, one which lists the latest versions of every known file (across all pathogens), and one which provides the version history for a single pathogen (only called when a modal opens). For the former would it be better to use data.nextstrain.org (cloudfronted) URLs? If so we can add a similar `replace()` to the server code.

Reply:
Probably in this function if we were to do so

https://github.com/nextstrain/nextstrain.org/blob/d8f3c5691e3befd87a674e899964689b439b4ad0/src/resourceIndex.js#L269-L277

Reply:
Yeah, I think we would want CloudFront for the unversioned URLs. Thanks for the pointer.